examples/game_of_life/pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = ["National Instruments"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-ni-measurement-plugin-sdk-service = {version = "^2.1.0-dev0", allow-prereleases = true}
+ni-measurement-plugin-sdk-service = {version = "^2.2.0-dev0", allow-prereleases = true}
```

Comment:
My understanding of the release process is to change all of these to ^2.1.0 (the newly released version) and remove the allow-prereleases. There's no reason to point to a new dev version that doesn't yet exist.

Reply:
This is why the Check Examples failed - 2.2.0-dev0 doesn't exist.